app/views/pages/_editor_guide_text.html.erb

```diff
@@ -80,14 +80,34 @@
         <figcaption>You can even add a caption using the HTML <code>figcaption</code> tag!</figcaption>
     </figure>
 
-    <%= render partial: "editor_guide_h3", locals: { id: "markdown-headers", title: "Headers" } %>
-    <p>Add a header to your post with this syntax:</p>
-    <pre># One '#' for a h1 header<br>## Two '#'s for a h2 header<br>...<br>###### Six '#'s for a h6 header</pre>
+    <%= render partial: "editor_guide_h3", locals: { id: "markdown-headers", title: "Headings" } %>
+    <p>Add a heading to your post with this syntax:</p>
```

Comment:
I've changed these references to 'heading' to remove any potential confusion in the distinction between 'headings' and 'headers'